src/components/category/category.js

```diff
@@ -1,50 +1,41 @@
-import category1 from '../../assets/grocery.jpg'
-import category2 from '../../assets/mobile.png'
-import category3 from '../../assets/black.jpeg'
-import category4 from '../../assets/headphone-1.jpeg'
-import category5 from '../../assets/home.png'
-import category6 from '../../assets/appliances_1.jpg'
 import {Link} from 'react-router-dom'
+import { useEffect } from 'react'
+import axios from 'axios'
+import { useCategory } from '../../filter-context/category-context'
+import {useFilter} from '../../filter-context/filter-context'
 export const Category=()=>{
+
+    const {category,setCategory}=useCategory()
+
+    const {state,dispatch}=useFilter()
+
+
+    useEffect(()=>{
+        homeCategory()
+    },[])
+
+    const homeCategory= async ()=>{
+        try {
+            
+            const response=await axios.get('/api/categories')
+            setCategory(response.data.categories);
+        } catch (error) {
+            console.log(error);
+        }
+    }
+
     return(
```

Comment:
This function name seems not according to what it is doing

Reply:
yes will change it